Vagrantfile

```diff
@@ -22,6 +23,12 @@ if options['github_token'].nil? || options['github_token'].to_s.length != 40
   exit
 end
 
+# check host ip
+if options['ip'] == "1"
```

Comment:
@cebe you are missing host ip(192.168.83.137) and guest ip(192.168.83.1). Guest always will be same as host but with `1` on the end